test/methods.jl

```diff
@@ -284,23 +285,36 @@ end
         # Quartimax is a special case of Oblimin
         test_equivalence(A, Quartimax(), Oblimin(gamma = 0, orthogonal = true); init)
 
-        # test that rotation result is identical published results by
-        # Bernaards & Jennrich (2005) within the reported accuracy of 7 digits
-        Ar = rotate(A, Quartimax(); init, atol = 1e-5)
-        Ar = round.(loadings(Ar), digits = 7)
-
-        pub = [
-            0.8987554 0.1948197
-            0.9339440 0.1297446
-            0.9021319 0.1038604
-            0.8765090 0.1712805
-            0.3155758 0.8764747
-            0.2511265 0.7734879
-            0.1980102 0.7146775
-            0.3078601 0.6593331
+        # test that rotation result is identical to GPArotation
+        Ar = rotate(A, Quartimax(); init, atol = 1e-7)
+
+        # loadings published in Bernaards & Jennrich (2005)
+        # within the reported accuracy of 7 digits
+        #pub = [
+        #    0.8987554 0.1948197
+        #    0.9339440 0.1297446
+        #    0.9021319 0.1038604
+        #    0.8765090 0.1712805
+        #    0.3155758 0.8764747
+        #    0.2511265 0.7734879
+        #    0.1980102 0.7146775
+        #    0.3078601 0.6593331
+        #]
+        # loadings obtained with GPArotation v2024.3
+        # quartimax(A, eps=1e-8, maxit=50000, randomStarts=10)
+        gpa = [
```

Comment:
Do you think it is a good idea to test against a battle tested package such as GPArotation for a given problem?

Reply:
I think it is, it could also be done more consistently for all the methods that the packages have in common.

One general consideration here is if we are comparing the rotation results against some reference, which has limited precision (e.g. the results from 2005 paper) -- that limited precision implies that the corresponding rotation may slightly violate the orthogonality constraints, and the corresponding reference criterion might be accidentally more optimal because the "point" is outside of the domain.
So ideally all reference rotations should contain as many digits after 0 as possible.